morpheus-data/morpheus_data/repository/model_repository.py

```diff
@@ -28,20 +28,28 @@ def create_model(cls, *, db: Session, model: MLModelCreate, categories: List[Mod
         return db_model
 
     @classmethod
-    def get_models(cls, *, db: Session, skip: int = 0, limit: int = 100) -> List[MLModel]:
-        return db.query(MLModel).filter(MLModel.is_active).offset(skip).limit(limit).all()
+    def get_models(cls, *, db: Session, skip: int = 0, limit: int = 100, only_active: bool = True) -> List[MLModel]:
+        if only_active:
+            return db.query(MLModel).filter(MLModel.is_active).offset(skip).limit(limit).all()
+        return db.query(MLModel).offset(skip).limit(limit).all()
 
     @classmethod
-    def get_model_by_id(cls, *, db: Session, model_id: UUID) -> MLModel:
-        return db.query(MLModel).filter(MLModel.id == model_id, MLModel.is_active).first()
+    def get_model_by_id(cls, *, db: Session, model_id: UUID, only_active: bool = True) -> MLModel:
```

Comment:
get_model_by_id should not have filters, it should always return a model regardless of the status